pkg/restapi/vc/operation/operations.go

```diff
@@ -697,6 +695,75 @@ func (o *Operation) retrieveCredentialHandler(rw http.ResponseWriter, req *http.
 	o.retrieveCredential(rw, profile, docURLs)
 }
 
+func (o *Operation) createDIDUniRegistrar(pr *ProfileRequest) (string, string, string, error) {
+	var opts []uniregistrar.CreateDIDOption
+
+	_, base58PubKey, err := o.kms.CreateKeySet()
+	if err != nil {
+		return "", "", "", err
+	}
+
+	_, recoveryPubKey, err := o.kms.CreateKeySet()
+	if err != nil {
+		return "", "", "", err
+	}
+
+	opts = append(opts,
+		uniregistrar.WithPublicKey(&didmethodoperation.PublicKey{
+			ID: pubKey1, Type: didclient.JWSVerificationKey2020,
+			Value:    base64.StdEncoding.EncodeToString(base58.Decode(base58PubKey)),
+			Encoding: didclient.PublicKeyEncodingJwk, Usage: []string{didclient.KeyUsageGeneral, didclient.KeyUsageOps}}),
+		uniregistrar.WithPublicKey(&didmethodoperation.PublicKey{
+			ID: recoveryKey1, Type: didclient.JWSVerificationKey2020,
+			Value:    base64.StdEncoding.EncodeToString(base58.Decode(recoveryPubKey)),
```

Comment:
we still need to have an ed25519 signature suite key available to sign VCs for the interop requirements.

Reply:
yes

Reply:
I will add it in next update for sidetree